update.php

```diff
@@ -373,7 +386,10 @@
 		} elseif ( $image === 'cli' ) {
 			// Replace tags inside the WP-CLI Dockerfile template.
 			if ( $config ) {
```

Comment:
There's no need for this condition anymore, since all PHP versions will have `$config` defined.